webpack.config.js

```diff
@@ -32,7 +33,7 @@ module.exports = {
         filename: productionBuild ? "librarie.min.js" : "librarie.js",
         path: __dirname + "/dist/" + version + "/",
         publicPath: "./dist/" + version,
-        libraryTarget: "var",
+        libraryTarget: "umd",
```

Comment:
http://tomasalabes.me/blog/_site/web-development/2016/04/30/Webpack-Series-Part-1.html 
Check this site...it is explained very clearly.